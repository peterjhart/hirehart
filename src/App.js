import React from "react";

function App() {
  return (
    <div className="header">
      <header className="text-center mb-3">
        <h1>Peter needs a job.</h1>
      </header>

      <div className="row mb-3">
        <div className="col text-center"></div>
        <div className="col text-center">
          <p>
            <strong>The source for this website is on github!</strong>
          </p>
          <p>
            <a
              href="https://github.com/peterjhart/hirehart"
              target="_blank"
              rel="noopener noreferrer"
            >
              GitHub
            </a>
          </p>
        </div>
        <div className="col text-center"></div>
      </div>
    </div>
  );
}

export default App;
