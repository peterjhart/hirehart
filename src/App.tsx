import { ReactElement } from 'react'
import { FileTextIcon, GithubIcon, MailIcon } from 'lucide-react'

export default function App(): ReactElement {
  return (
    <div className="header">
      <header className="text-center mb-3">
        <h1>Peter needs a job.</h1>
      </header>

      <div className="row">
        <div className="col-md-4 mb-3 mb-md-0">
          <div className="card py-4 h-100">
            <div className="card-body text-center">
              <FileTextIcon className="text-primary mb-2" />
              <h4 className="text-uppercase m-0">Résumé</h4>
              <hr className="my-4" />
              <div className="small text-black-50">
                <a href="/assets/peter%20j.%20hart%20resume%202024.pdf">PDF</a>
              </div>
            </div>
          </div>
        </div>
        <div className="col-md-4 mb-3 mb-md-0">
          <div className="card py-4 h-100">
            <div className="card-body text-center">
              <GithubIcon className="text-primary mb-2"></GithubIcon>
              <h4 className="text-uppercase m-0">GitHub</h4>
              <hr className="my-4" />
              <div className="small text-black-50">
                See the source for this website on&nbsp;
                <a
                  href="https://github.com/peterjhart/hirehart"
                  target="_blank"
                  rel="noopener noreferrer"
                >
                  GitHub
                </a>
              </div>
            </div>
          </div>
        </div>
        <div className="col-md-4 mb-3 mb-md-0">
          <div className="card py-4 h-100">
            <div className="card-body text-center">
              <MailIcon className="text-primary mb-2" />
              <h4 className="text-uppercase m-0">Contact</h4>
              <hr className="my-4" />
              <div className="small text-black-50">
                <a href="mailto:peter@hirehart.com">peter@hirehart.com</a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  )
}
