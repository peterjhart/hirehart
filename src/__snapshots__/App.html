<div>
  <div
    class="py-8"
  >
    <header
      class="text-center mb-3"
    >
      <h1
        class="text-8xl mb-8"
      >
        Hire Peter Hart
      </h1>
    </header>
    <div
      class="grid xs:grid-cols-1 md:grid-cols-3 gap-4 mx-12"
    >
      <div
        class="mb-3 p-4 border rounded-lg h-full"
      >
        <div
          class="flex justify-center"
        >
          <svg
            class="lucide lucide-file-text h-12 w-12 mb-2 text-cyan-600"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M15 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V7Z"
            />
            <path
              d="M14 2v4a2 2 0 0 0 2 2h4"
            />
            <path
              d="M10 9H8"
            />
            <path
              d="M16 13H8"
            />
            <path
              d="M16 17H8"
            />
          </svg>
        </div>
        <h2
          class="text-center text-3xl text-cyan-600"
        >
          Résumé
        </h2>
        <hr
          class="my-4"
        />
        <div
          class="text-center text-balance"
        >
          <a
            class="text-cyan-600 underline underline-offset-4"
            href="/assets/peter%20j.%20hart%20resume%202024.pdf"
          >
            PDF
          </a>
        </div>
      </div>
      <div
        class="mb-3 p-4 border rounded-lg h-full"
      >
        <div
          class="flex justify-center"
        >
          <svg
            class="lucide lucide-square-code h-12 w-12 mb-2 text-cyan-600"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M10 9.5 8 12l2 2.5"
            />
            <path
              d="m14 9.5 2 2.5-2 2.5"
            />
            <rect
              height="18"
              rx="2"
              width="18"
              x="3"
              y="3"
            />
          </svg>
        </div>
        <h4
          class="text-center text-3xl text-cyan-600"
        >
          GitHub
        </h4>
        <hr
          class="my-4"
        />
        <div
          class="text-center text-balance"
        >
          See the source for this website on 
          <a
            class="text-cyan-600 underline underline-offset-4"
            href="https://github.com/peterjhart/hirehart"
            rel="noopener noreferrer"
            target="_blank"
          >
            GitHub
          </a>
        </div>
      </div>
      <div
        class="mb-3 p-4 border rounded-lg h-full"
      >
        <div
          class="flex justify-center"
        >
          <svg
            class="lucide lucide-mail h-12 w-12 mb-2 text-cyan-600"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <rect
              height="16"
              rx="2"
              width="20"
              x="2"
              y="4"
            />
            <path
              d="m22 7-8.97 5.7a1.94 1.94 0 0 1-2.06 0L2 7"
            />
          </svg>
        </div>
        <h4
          class="text-center text-3xl text-cyan-600"
        >
          Contact
        </h4>
        <hr
          class="my-4"
        />
        <div
          class="text-center text-balance"
        >
          <a
            class="text-cyan-600 underline underline-offset-4"
            href="mailto:peter@hirehart.com"
          >
            peter@hirehart.com
          </a>
        </div>
      </div>
    </div>
  </div>
</div>